BangazonAPI/Models/ProductTypes.cs

```diff
@@ -0,0 +1,17 @@
+/*
+Author: Klaus Hardt
+Purpose: Model for Product Types including gets/sets for all db columns and List. Also Ricky sucks.
+*/
+using System.Collections.Generic;
+
+namespace BangazonAPI.Data
+{
+    public class ProductTypes
```

Comment:
This should be `ProductType` not `ProductTypes`

Reply:
woooooooow